/**:
  ros__parameters:
    gnss_enabled: true
    ndt_enabled: true
    ekf_enabled: true
    stop_check_enabled: true
